src/test/groovy/org/prebid/server/functional/model/response/auction/Privacy.groovy

```diff
@@ -0,0 +1,16 @@
+package org.prebid.server.functional.model.response.auction
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import groovy.transform.ToString
+import org.prebid.server.functional.model.bidder.BidderName
+
+@ToString(includeNames = true, ignoreNulls = true)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy)
+class Privacy {
```

Comment:
Better switch places in names: name this class DebugPrivacy and other just Privacy.